libs/tools/src/generators/helm-library/generator.ts

```diff
@@ -0,0 +1,159 @@
+import { VERSION } from '@angular/core';
+import { libraryGenerator, UnitTestRunner } from '@nx/angular/generators';
+import {
+	formatFiles,
+	joinPathFragments,
+	names,
+	readProjectConfiguration,
+	Tree,
+	updateJson,
+	updateProjectConfiguration,
+} from '@nx/devkit';
+import {} from 'semver';
```

Comment:
minor: I thought Prettier removes unused imports automatically?
```suggestion
```
Hmm, maybe that's from `prettier-plugin-organize-imports` only.

Reply:
We can probably also remove semver from the package.json since it looks like it ended up being unnecessary here.